File: ProjetPI/src/ProjectBundle/Entity/Refugies.php
<?php

namespace ProjectBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
<<<<<<< HEAD
use Symfony\Component\Validator\Constraints as Assert;
use Symfony\Component\Validator\Mapping\ClassMetadata;
=======
>>>>>>> af282b06661706d128dc0ce87d0d0196d5239e87

/**
 * Refugies
 *
 * @ORM\Table(name="refugies")
 * @ORM\Entity(repositoryClass="ProjectBundle\Repository\RefugiesRepository")
 */
class Refugies
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_ref", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id_ref;
    /**
     * @var string
     *
     * @ORM\Column(name="pays_ref", type="string", length=255)
     */
    private $paysRef;

    /**
     * @return int
     */
    /**
     *
     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_membre", referencedColumnName="id", nullable=true)
     * })
     */
    private $id_membre;

    /**
     * @return mixed
     */
    public function getIdMembre()
    {
        return $this->id_membre;
    }

    /**
     * @param mixed $id_membre
     */
    public function setIdMembre($id_membre)
    {
        $this->id_membre = $id_membre;
    }

<<<<<<< HEAD
    public function __toString()
    {
        return $this->nomRef;
    }

=======
>>>>>>> af282b06661706d128dc0ce87d0d0196d5239e87
    /**
     * @param int $ageRef
     */
    public function setAgeRef($ageRef)
    {
        $this->ageRef = $ageRef;
    }

    /**
     * @param string $etatRef
     */
    public function setEtatRef($etatRef)
    {
        $this->etatRef = $etatRef;
    }


    public function getAgeRef()
    {
        return $this->ageRef;
    }

    /**
     * @return int
     */
    public function getIdRef()
    {
        return $this->id_ref;
    }

    /**
     * @param int $id_ref
     */
    public function setIdRef($id_ref)
    {
        $this->id_ref = $id_ref;
    }

    /**
     * @return string
     */
    public function getPaysRef()
    {
        return $this->paysRef;
    }

    /**
     * @param string $paysRef
     */
    public function setPaysRef($paysRef)
    {
        $this->paysRef = $paysRef;
    }

    /**
     * @return string
     */
    public function getNomRef()
    {
        return $this->nomRef;
    }

    /**
     * @param string $nomRef
     */
    public function setNomRef($nomRef)
    {
        $this->nomRef = $nomRef;
    }

    /**
     * @return string
     */
    public function getPrenomRef()
    {
        return $this->prenomRef;
    }

    /**
     * @param string $prenomRef
     */
    public function setPrenomRef($prenomRef)
    {
        $this->prenomRef = $prenomRef;
    }

    /**
     * @return string
     */
    public function getGenderRef()
    {
        return $this->genderRef;
    }

    /**
     * @param string $genderRef
     */
    public function setGenderRef($genderRef)
    {
        $this->genderRef = $genderRef;
    }

    /**
     * @return string
     */
    public function getImage()
    {
        return $this->image;
    }

    /**
     * @param string $image
     */
    public function setImage($image)
    {
        $this->image = $image;
    }

    /**
     * @return string
     */
    public function getEtatRef()
    {
        return $this->etatRef;
    }

    /**
     * @var string
<<<<<<< HEAD
     *@Assert\Type("alpha",
     * message="ce champs doit etre de type string ")
=======
     *
>>>>>>> af282b06661706d128dc0ce87d0d0196d5239e87
     * @ORM\Column(name="etat_ref", type="string", length=255)
     */
    private $etatRef;

    /**
     * @var string
     *
<<<<<<< HEAD
     * @Assert\Type("alpha",
     * message="ce champs doit etre de type string ")
     *
=======
>>>>>>> af282b06661706d128dc0ce87d0d0196d5239e87
     * @ORM\Column(name="nom_ref", type="string", length=255)
     */
    private $nomRef;

    /**
<<<<<<< HEAD
     * @Assert\Type("alpha",
     * message="ce champs doit etre de type string ")
     *
=======
     * @var string
>>>>>>> af282b06661706d128dc0ce87d0d0196d5239e87
     *
     * @ORM\Column(name="prenom_ref", type="string", length=255)
     */
    private $prenomRef;

    /**
     * @var int
     *
<<<<<<< HEAD
     *
     * @Assert\GreaterThan(value=0,
     * message="Le nombre doit etre supérieur à 0 "
     *)
     *
     *
=======
>>>>>>> af282b06661706d128dc0ce87d0d0196d5239e87
     * @ORM\Column(name="age_ref", type="integer")
     */
    private $ageRef;

    /**
     * @var string
     *
<<<<<<< HEAD
     *@Assert\Type("alpha",
     * message="ce champs doit etre de type string ")
     *
=======
>>>>>>> af282b06661706d128dc0ce87d0d0196d5239e87
     * @ORM\Column(name="gender_ref", type="string", length=255)
     */
    private $genderRef;

    /**
     * @var string
     *
     * @ORM\Column(name="image", type="string", length=255,nullable=true)
     */
    private $image;

<<<<<<< HEAD
    public static function loadValidatorMetadataAction(ClassMetadata $metadata)
    {

        $metadata->addPropertyConstraint('prenomRef', new Assert\Type('string'));

        $metadata->addPropertyConstraint('ageRef', new Assert\Type([
            'type' => 'integer',
            'message' => 'The value {{ value }} is not a valid {{ type }}.',
        ]));
    }
=======
>>>>>>> af282b06661706d128dc0ce87d0d0196d5239e87


}

File: ProjetPI/src/entities/Refugies.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

import java.util.Objects;

/**
 *
 * @author dEll
 */
public class Refugies {
    
    private int id_ref ;
    private int id_membere ;
    private String pays_ref ;
    private String etat_ref ;
    private String nom_ref ;
    private String prenom_ref ;
    private int age_ref ;
    private String gender_ref ;
    private String image ;

    public Refugies(String pays_ref, String nom_ref, String prenom_ref, int age_ref, String gender_ref) {
        this.pays_ref = pays_ref;
        this.nom_ref = nom_ref;
        this.prenom_ref = prenom_ref;
        this.age_ref = age_ref;
        this.gender_ref = gender_ref;
    }

    public Refugies(int id_ref, int id_membere, String pays_ref, String etat_ref, String nom_ref, String prenom_ref, int age_ref, String gender_ref, String image) {
        this.id_ref = id_ref;
        this.id_membere = id_membere;
        this.pays_ref = pays_ref;
        this.etat_ref = etat_ref;
        this.nom_ref = nom_ref;
        this.prenom_ref = prenom_ref;
        this.age_ref = age_ref;
        this.gender_ref = gender_ref;
        this.image = image;
    }

    public Refugies(String pays_ref, String etat_ref, String nom_ref, String prenom_ref, int age_ref, String gender_ref, String image) {
        this.pays_ref = pays_ref;
        this.etat_ref = etat_ref;
        this.nom_ref = nom_ref;
        this.prenom_ref = prenom_ref;
        this.age_ref = age_ref;
        this.gender_ref = gender_ref;
        this.image = image;
    }

    public Refugies(String pays_ref, String nom_ref, String prenom_ref, int age_ref, String gender_ref, String image) {
        this.pays_ref = pays_ref;
        this.nom_ref = nom_ref;
        this.prenom_ref = prenom_ref;
        this.age_ref = age_ref;
        this.gender_ref = gender_ref;
        this.image = image;
    }

    
    
    
    public Refugies() {
    }

    public int getId_ref() {
        return id_ref;
    }

    public void setId_ref(int id_ref) {
        this.id_ref = id_ref;
    }

    public int getId_membere() {
        return id_membere;
    }

    public void setId_membere(int id_membere) {
        this.id_membere = id_membere;
    }

    public String getPays_ref() {
        return pays_ref;
    }

    public void setPays_ref(String pays_ref) {
        this.pays_ref = pays_ref;
    }

    public String getEtat_ref() {
        return etat_ref;
    }

    public void setEtat_ref(String etat_ref) {
        this.etat_ref = etat_ref;
    }

    public String getNom_ref() {
        return nom_ref;
    }

    public void setNom_ref(String nom_ref) {
        this.nom_ref = nom_ref;
    }

    public String getPrenom_ref() {
        return prenom_ref;
    }

    public void setPrenom_ref(String prenom_ref) {
        this.prenom_ref = prenom_ref;
    }

    public int getAge_ref() {
        return age_ref;
    }

    public void setAge_ref(int age_ref) {
        this.age_ref = age_ref;
    }

    public String getGender_ref() {
        return gender_ref;
    }

    public void setGender_ref(String gender_ref) {
        this.gender_ref = gender_ref;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    @Override
    public int hashCode() {
        int hash = 5;
        hash = 23 * hash + this.id_ref;
        hash = 23 * hash + this.id_membere;
        hash = 23 * hash + Objects.hashCode(this.pays_ref);
        hash = 23 * hash + Objects.hashCode(this.etat_ref);
        hash = 23 * hash + Objects.hashCode(this.nom_ref);
        hash = 23 * hash + Objects.hashCode(this.prenom_ref);
        hash = 23 * hash + this.age_ref;
        hash = 23 * hash + Objects.hashCode(this.gender_ref);
        hash = 23 * hash + Objects.hashCode(this.image);
        return hash;
    }

    @Override
    public boolean equals(Object obj) {
        if (this == obj) {
            return true;
        }
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final Refugies other = (Refugies) obj;
        if (this.id_ref != other.id_ref) {
            return false;
        }
        if (this.id_membere != other.id_membere) {
            return false;
        }
        if (this.age_ref != other.age_ref) {
            return false;
        }
        if (!Objects.equals(this.pays_ref, other.pays_ref)) {
            return false;
        }
        if (!Objects.equals(this.etat_ref, other.etat_ref)) {
            return false;
        }
        if (!Objects.equals(this.nom_ref, other.nom_ref)) {
            return false;
        }
        if (!Objects.equals(this.prenom_ref, other.prenom_ref)) {
            return false;
        }
        if (!Objects.equals(this.gender_ref, other.gender_ref)) {
            return false;
        }
        if (!Objects.equals(this.image, other.image)) {
            return false;
        }
        return true;
    }

    @Override
    public String toString() {
        return "Refugies{" + "id_ref=" + id_ref + ", id_membere=" + id_membere + ", pays_ref=" + pays_ref + ", etat_ref=" + etat_ref + ", nom_ref=" + nom_ref + ", prenom_ref=" + prenom_ref + ", age_ref=" + age_ref + ", gender_ref=" + gender_ref + ", image=" + image + '}';
    }
    
    
    
    
}
